### ISSUE TITLE
    bs, _, ny, nx = x[i].shape  #x(bs, 255, 20,20) to x(bs,3,20,20, 85)   How  is x  transformed from 4-dim to 5-dim? 

### ISSUE THREAD

**Issue by ShirleyHe2020** [yl305237731/flexible-yolov5#14]
in  od/models/head/yolo.py /

    def forward(self, x):
        z = [] # inference output
        self.training |= self.export
        for i in range(self.nl):
            x[i] = self.m[i](x[i])
            bs, _, ny, nx = x[i].shape  #x(bs, 255, 20,20) to x(bs,3,20,20, 85)
            x[i] = x[i].view(bs, self.na, self.no, ny, nx).permute(0, 1, 3, 4, 2).contiguous()

**Comment by yl305237731** [comment]
it seems that you are new to pytorch.....
